tests/luaL_buffsub_test.cc

```diff
@@ -27,13 +27,13 @@ LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 	if (L == NULL)
 		return 0;
 
-	luaL_Buffer buf;
 	auto str = fdp.ConsumeRandomLengthString(size);
 	size_t buf_size = str.length() + 1;
-	char *s = luaL_buffinitsize(L, &buf, buf_size);
-	memcpy(s, str.c_str(), buf_size);
-	luaL_pushresultsize(&buf, buf_size);
-	int8_t n = fdp.ConsumeIntegralInRange<int8_t>(0, buf_size);
+	int n = fdp.ConsumeIntegralInRange<int>(0, buf_size);
```

Comment:
@ligurio 
Maybe it would be better to set template type to the type of `buf_size`. It will be converted during the call to `int` and it can become negative, so the `abort` may still appear.

Reply:
Or at least it makes sense to make the template type unsigned.

Reply:
Replaced with `uint8_t`.